pkg/pods/hoist_executable_test.go

```diff
@@ -0,0 +1,47 @@
+package pods
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"os/user"
+	"path"
+	"strings"
+	"testing"
+
+	. "github.com/anthonybishopric/gotcha"
+)
+
+func TestExecutableWritesValidScript(t *testing.T) {
+	user, err := user.Current()
+	Assert(t).IsNil(err, "test setup failure - should not have failed to get current user")
+	envdir, err := ioutil.TempDir("", "envdir")
```

Comment:
I'm also making a (maybe dumb) assumption that the next line, which is also a temp directory creation, would almost definitely have an error if the previous line had an error.